r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -159,31 +158,21 @@ private void resolveDefinition(Session session, PlanDefinition.PlanDefinitionAct
                 IdType id = new IdType(definition);
                 CarePlan plan;
                 try {
-                    plan = applyPlanDefinition(id,
-                            session.getPatientId(),
-                            session.getEncounterId(),
-                            session.getPractitionerId(),
-                            session.getOrganizationId(),
-                            session.getUserType(),
-                            session.getUserLanguage(),
-                            session.getUserTaskContext(),
-                            session.getSetting(),
+                    plan = applyPlanDefinition(id, session.getPatientId(), session.getEncounterId(),
+                            session.getPractitionerId(), session.getOrganizationId(), session.getUserType(),
+                            session.getUserLanguage(), session.getUserTaskContext(), session.getSetting(),
                             session.getSettingContext());
 
                     if (plan.getId() == null) {
                         plan.setId(UUID.randomUUID().toString());
                     }
 
                     // Add an action to the request group which points to this CarePlan
-                    session.getRequestGroupBuilder()
-                        .buildContained(plan)
-                        .addAction(new RequestGroupActionBuilder()
-                            .buildResource(new Reference("#" + plan.getId()))
-                            .build()
-                        );
-
-                    for(CanonicalType c: plan.getInstantiatesCanonical())
-                    {
+                    session.getRequestGroupBuilder().buildContained(plan)
```

Comment:
I think these formatting changes make this code less readable.  Without a common checkstyle configuration in place I'm wary of having all our different editor/IDE opinions battling it out.

**Before:**
![Screen Shot 2020-10-07 at 12 13 28 PM](https://user-images.githubusercontent.com/41651655/95358046-9b62eb80-0896-11eb-9bf0-8d114ac94fb5.png)

**After:**

![Screen Shot 2020-10-07 at 12 15 14 PM](https://user-images.githubusercontent.com/41651655/95358192-c6e5d600-0896-11eb-89f2-d6e1726d56bc.png)